kinto_http/__init__.py

```diff
@@ -213,6 +215,9 @@ def create_bucket(self, bucket=None, data=None, permissions=None,
                                               safe=safe)
         headers = DO_NOT_OVERWRITE if safe else None
         endpoint = self.get_endpoint('bucket', bucket=bucket)
+
+        logger.info("Create bucket %r with data %r" % (bucket, data))
```

Comment:
@leplatrem so should I completely remove data from `info` level messages and just keep the name of the object being created/updated/get-ed/deleted?